crates/biome_cli/tests/snapshots/main_commands_ci/ci_formatter_linter_organize_imports.snap

```diff
@@ -52,10 +51,11 @@ file.js assist ━━━━━━━━━━━━━━━━━━━━━
     1 1 │   
     2   │ - import·{·B,·C·}·from·"b.js"
     3   │ - import·A·from·"a.js"
-      2 │ + import·A·from·"a.js"
-      3 │ + import·{·B,·C·}·from·"b.js"
-    4 4 │   
-    5 5 │   
+      2 │ + 
```

Comment:
Why does this insert an extra newline? Is that also due to trivia handling?

Reply:
This looks like a bug.